### PR TITLE
trying to make the output more readable

### PR DESCRIPTION
**Pull request by jspaaks**
## Description

This PR adds printing the output from ort wrapped in a collapsible group so you get a cleaner overview of the progress in the logs, like so:

image

More elaborate example: https://github.com/NLeSC/tortellini-on-rsd/runs/2720033149?check_suite_focus=true

Refs: none

* * *

## Before you get started

- [x] I read and I followed [contributing guidelines](https://github.com/tortellini-tools/action/blob/main/CONTRIBUTING.md)
- [ ] [☝ Create an issue](https://github.com/NLeSC/licenseguard/issues/new/choose) to discuss what you are going to do

## Checklist

This section should be filled in by the creator of this pull request to make sure the pull request is ready to review.

- [x] This pull request has a descriptive title
- [x] Code is written according to the code quality guidelines
- [x] Documentation if applicable is available
- [x] Tests run successfully
- [x] All checks below this pull request were successful

## Instructions to review the pull request

Open question: Not sure if using the sync `core.startGroup()` and `core.endGroup()` is correct, maybe we should swithc to async `core.group()`

### CODE REVIEW

File: src/check.ts [hunk 1]
```diff
@@ -24,14 +24,27 @@ export async function check_urls(
...
     }
 }
+
+function print_collapsable_end(): void {
+    console.log('##endgroup')
+}
+
+function print_collapsable_start(i: number, n: number, gitrepo: GitRepo): void {
+    console.log(`##group${i}/${n}: ${gitrepo.owner}/${gitrepo.repo}`)
+}
```
Comment (sverhoeven) [inline src/check.ts]: Please use https://github.com/actions/toolkit/tree/main/packages/core#logging methods instead of rolling your own

Comment (jspaaks) [reply src/check.ts]: switched, thanks for the suggestion

**Review by sverhoeven — APPROVED** [review]
Looks much nicer

We should not forget to change back to `main` branch in `.github/workflows/tortellini.yml` and https://github.com/NLeSC/tortellini-on-rsd/blob/main/.github/workflows/tortellini.yml

File: .github/workflows/tortellini.yml [hunk 2]
```diff
@@ -20,7 +20,7 @@ jobs:
       - run: |
           echo 'https://github.com/tortellini-tools/action' > urls.txt
           echo 'https://github.com/fair-software/howfairis' >> urls.txt
-      - uses: tortellini-tools/action@main
+      - uses: tortellini-tools/action@neater-output
```
Comment (jspaaks) [inline .github/workflows/tortellini.yml]:
```suggestion
      - uses: tortellini-tools/action@main
```

Comment (jspaaks) [reply .github/workflows/tortellini.yml]: reverted to main branch

**Comment by jspaaks** [comment]
Thanks for reviewing, I reverted the workflow back to main as you suggested

**Pull request merged**